echemdb/data/cv/descriptor.py

```diff
@@ -104,10 +104,12 @@ def __getattr__(self, name):
 
         """
         name = name.replace('_', ' ')
-        if name in self._descriptor:
-            return Descriptor(self._descriptor[name])
-
-        raise AttributeError(f"Descriptor has no entry {name}. Did you mean one of {[key.replace(' ', '_') for key in self._descriptor.keys()]}?")
+        
```

Comment:
I  think you want to revert this change. This should be fixed by now (and returning `None` here is not alowed.)

Reply:
my mistake. The old PR was merged in this one too.